packages/shared/src/widgets/layouts/main-layout/header.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import { usePathname } from 'next/navigation';
+
+import { BREADCRUMB_ROUTES } from '@it-diots/shared/constants/page-routes';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbList,
+  BreadcrumbPage,
+  BreadcrumbSeparator,
+  Separator,
+  SidebarTrigger,
+} from '@it-diots/shared/ui';
+
+export const Header = () => {
```

Comment:
나머지 컴포넌트들이 export function으로 내보내지고 있어서 통일하면 좋을것 같습니다!

```suggestion
export function Header() {
```

Reply:
확인했습니다! 앞으로 해당 컨벤션으로 맞추겠습니다.